docs/source/authentication.rst

```diff
@@ -134,58 +134,69 @@ The following is a list of auth backends for the community edition to help get t
 * `Flat File <https://github.com/StackStorm/st2-auth-backend-flat-file>`_
 * `OpenStack Keystone <https://github.com/StackStorm/st2-auth-backend-keystone>`_
 
-LDAP (Enterprise Edition)
--------------------------
-.. note::
-
-  Prior to StackStorm 3.3, Extreme Networks provided a commercial version(Extreme Workflow Composer) of the StackStorm automation platform including an LDAP auth backend.
```

Comment:
Yes, same as in RBAC PR, I think it worth keeping this note in place and also mentioning that `LDAP` is now available in StackStorm OSS since version `3.4`.

```rst
.. note::

  Prior to StackStorm 3.3, Extreme Networks provided a commercial version(Extreme Workflow Composer) of the StackStorm automation platform including an LDAP auth backend. As these enterprise features were donated to the Linux Foundation, LDAP is now available in StackStorm Open Source since 3.4.
```